src/corelibs/U2Gui/src/options_panel/OPWidgetFactory.cpp

```diff
@@ -40,6 +51,16 @@ bool OPFactoryFilterVisitor::atLeastOneAlphabetPass(DNAAlphabetType factoryAlpha
     return false;
 }
 
+bool OPFactoryFilterVisitor::atLeastOneDnaPass() const {
```

Comment:
Do we have any use-case for this method in this PR?
It looks like it computes something static - that can be computed in the constructor.

Reply:
No, we don't, removed